src/common/input/handler.rs

```diff
@@ -276,10 +263,12 @@ pub enum InputMode {
     Exiting,
 }
 
+/// Represents the help message that is printed in the status bar, indicating
+/// the current [`InputMode`], whether that mode is persistent, and what the
```

Comment:
"whether that mode is persistent" - not anymore :) (but no biggie, we can remove this on the next pass)